frontend/src/apis/run/api.ts

```diff
@@ -1054,6 +1096,24 @@ export const RunServiceApiFp = function(configuration?: Configuration) {
                 });
             };
         },
+        /**
+         * 
+         * @param {string} run_id 
+         * @param {*} [options] Override http request option.
+         * @throws {RequiredError}
+         */
+        retryRun(run_id: string, options?: any): (fetch?: FetchAPI, basePath?: string) => Promise<any> {
```

Comment:
Revert this, it's not related to this PR. The functions below as well

Reply:
yeah this pr covers the change
https://github.com/kubeflow/pipelines/pull/1782